docs/pages/academy/6-analytics/3-graph-analytics.md

```diff
@@ -13,18 +13,18 @@ KB: academy
 
 At this point in the academy it should be somewhat clear that GRAKN is not a graph database. So why a lesson about graph analytics?
 
-Well the answer is that an entity-relationship conceptual model (thus a GRAKN knowledge base) can actually be represented in the form of a graph.
+Well the answer is that an entity-relationship conceptual model (thus a GRAKN knowledge graph) can actually be represented in the form of a graph.
 
 And if you have a graph, then it is often useful to be able to do perform graph analytics.
 
 
 ## A few words before going on
 As you will see, it is quite straightforward to perform graph analytics with Grakn. There are two caveats though:
 
-  1. GRAKN is not a graph database, so to forse a knowledge based with a underlying hypergraph structure into a graph in order to perform graph analytics requires some getting used to.
+  1. GRAKN is not a graph database, so to forse a knowledge graphd with a underlying hypergraph structure into a graph in order to perform graph analytics requires some getting used to.
```

Comment:
:goat:, also FORSE

Reply:
Hahahahha fixed in https://github.com/graknlabs/grakn/pull/2681/commits/8532f9fbb7d8c3bd3d12d1239d37759be0efea0f